multik-api/src/main/kotlin/org/jetbrains/kotlinx/multik/api/LinAlg.kt

```diff
@@ -24,7 +24,7 @@ public interface LinAlg {
 
     public fun <T : Number> inv(mat: MultiArray<T, D2>): NDArray<T, D2>
 
-    public fun <T : Number, D: Dim2> solve(a: MultiArray<T, D2>, b: MultiArray<T, D>): NDArray<T, D>
+    public fun <T : Number, D: Dim2> solve(a: MultiArray<T, D2>, b: MultiArray<T, D>): NDArray<Double, D>
```

Comment:
Why Double?

Reply:
It's rare case when solution of a system with type T entries is also of type T. For example unclear what should return `public fun <T : Number, D: Dim2> solve(a: MultiArray<T, D2>, b: MultiArray<T, D>): NDArray<T, D>` in case
```
solve(a = mk.ndarray(mk[mk[1]]), b = mk.ndarray(mk[2]))
```

Reply:
Revert changes please.